notebooks/optimal-stopping-point/osp_model.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>Is there a purpose of filtering on this particular test? <code>operator.Run multi-stage test e2e-aws-upgrade - e2e-aws-upgrade-openshift-e2e-test container test</code>.  Instead of hardcoding this throughout notebook, is it possible to treat it as a parameter/variable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/445/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>We chose a random test for this notebook to show how the optimal stopping point is predicted. We were planning on making it a variable/parameter when we turn this model into a service. Do you think that would work?</p>